app/src/main/java/com/theopensourcefamily/chessclock/modules/schedulers.kt

```diff
@@ -0,0 +1,13 @@
+package com.theopensourcefamily.chessclock.modules
+
+import io.reactivex.Scheduler
+import io.reactivex.android.schedulers.AndroidSchedulers
+import org.koin.dsl.module
+
+val schedulers = module {
+
+  single { provideMainScheduler() }
+
+}
+
+fun provideMainScheduler(): Scheduler? = AndroidSchedulers.mainThread()
```

Comment:
This function doesn't need to be nullable (actually the presenter is constructed with a non nullable scheduler).
I think we could avoid creating this function until it's needed, and just provide directly the Scheduler:
`  single { AndroidSchedulers.mainThread() } `.
What do you think? 😄

Reply:
Sure. My mistake.